wrappers/wrapper.rb

```diff
@@ -1421,11 +1418,12 @@ def simplify_service_setup_duration_and_vehicle_setup_modifiers(vrp, solution =
 
             next if setup_duration.zero?
 
-            time_correction = coef_setup * setup_duration + additional_setup
+            # The vehicle adjustement is performed by vrp_result
+            stop.activity.setup_duration = setup_duration
+            travel_time_correction = stop.activity.setup_duration_on(vehicle)
```

Comment:
Just above there is `setup_duration = stop.activity[:simplified_setup_duration].to_i` I don't know why I did the rounding there but it should be done here in fact (because the matrix is modified as such so rewind/patch should of the inverse):

```suggestion
            travel_time_correction = stop.activity.setup_duration_on(vehicle).to_i
```